.github/workflows/ffigen.yml

```diff
@@ -28,7 +28,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [3.2.0]
+        sdk: [dev]
+#        sdk: [3.2.0]
```

Comment:
Maybe update the commented stuff to 3.3.0, and file an issue to update this when 3.3.0 beta is released.

Reply:
Done - #876.